src/formElementPlugins/search/src/ApplicationView.tsx

```diff
@@ -126,27 +136,10 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
     })
   }
 
-  type ResultsField = 'title' | 'description'
-  // If a displayFormat is not defined, results will be displayed some combination
-  // of the first two fields in the object
-  const getDefaultString = (result: any, fieldType: ResultsField = 'title') => {
-    if (!(result instanceof Object)) return result
-    const fields = Object.keys(result)
-    switch (fields.length) {
-      case 0:
-        return fieldType === 'title' ? strings.MESSAGE_NO_RESULTS : ''
-      case 1:
-        return fieldType === 'title' ? fields[0] : result[fields[0]]
-      default:
-        return fieldType === 'title'
-          ? `${fields[0]}: ${result[fields[0]]}`
-          : `${fields[1]}: ${result[fields[1]]}`
-    }
-  }
```

Comment:
Just moved this outside component so it can be called from the Display component as well